### DISCUSSION TITLE
Port Forwarding

### DISCUSSION THREAD

**Discussion by Sacrachan** [RaspAP/raspap-webgui#1223] [Q&A]
Hi guys,
recently I needed the Port 80 on my RaspberryPi 4 which has been used by RaspAp. To free that port I went into the settings of RaspAp and changed that port from 80 to 420. Now I have the problem that if I want to access the web page of RaspAp through the local IP address of my Pi I can only access the start page and everything that is behind the "/" of the IP address I get a "The requested URL was not found on this server." error from Apache. If I want to access the webpage through port 420 I just get the "That website can't be reached" error from Google. Does anyone of you know if there is a way to fix this error without a clean install of RaspAp? Or can I change that port with a file on my raspberry pi?

**Comment by billz** [comment]
RaspAP's port options modify lighttpd's `server.port` setting. It doesn't know anything about Apache. 
Are you running Apache alongside lighttpd? If so, try looking at their [official docs](https://httpd.apache.org/docs/2.4/en/bind.html).

Do you want to run a site other than RaspAP at the web root? It's possible to change this without a reinstall, if you're comfortable making manual changes. See https://docs.raspap.com/manual/#create-the-web-application

**Reply by Sacrachan** [reply]
Thanks for your reply,
I have tried to change the server .port setting back to 80 but it didn't help. I tried to reboot my Pi to see if the change worked but it didn't. The applied changes in Lighttpd's config file instead stayed at 80. 

I wanted to install Nextcloud on my Pi but after a few minutes I had too many errors and I don't need a Nextcloud server anymore. Now I also don't need port 80 anymore and I want a working web GUI for RaspAp back as it was before.

**Reply by billz** [reply]
You have two options:
1) reinstall RaspAP on a fresh SD (recommended).
2) modify your lighttpd.conf to match [this one](https://gist.github.com/billz/f21156ea74e574b446e7ad5ee081e5af).

**Reply by Sacrachan** [reply]
Well...
I tried to modify my Lighttpd.conf file by copying and pasting but it didn't work. 
If I try to reinstall RaspAp, do I need a fresh install of raspbian OS or does it also work if I just uninstall it on my current OS and reinstall it? The reason why I don't want to reset my Pi is because of the number of files I have stored on it and I have many programs installed on it.

**Reply by billz** [reply]
Restart lighttpd for the changes to take effect: `sudo systemctl restart lighttpd.service`

Otherwise, you can run RaspAP's uninstaller and install it again.
